### ISSUE TITLE
Acf Image

### ISSUE THREAD

**Issue by lupetalo** [jesperbjerke/wa-fronted#3]
Acf Image field, when url is selected as storage method, are not being updated after change, displays only a url. Only after page refresh picture is visible again.

**Comment by jesperbjerke** [comment]
Have you set the "output" and "output_to" parameters in your config array?

**Comment by lupetalo** [comment]
No, but it is saved as url, not array.

**Comment by jesperbjerke** [comment]
Ah, I see.

**Comment by jesperbjerke** [comment]
This will probably generate more trouble than easing configuration. Setting to 'wontfix' for now.
